cmd/virtual-kubelet/root/flag.go

```diff
@@ -72,6 +72,8 @@ func InstallFlags(flags *pflag.FlagSet, o *Opts) {
 	flags.BoolVar(&o.EnableStorage, "enable-storage", false, "Enable the Liqo storage reflection")
 	flags.StringVar(&o.VirtualStorageClassName, "virtual-storage-class-name", "liqo", "Name of the virtual storage class")
 	flags.StringVar(&o.RemoteRealStorageClassName, "remote-real-storage-class-name", "", "Name of the real storage class to use for the actual volumes")
+	flags.StringVar(&o.HostAPIServerHost, "host-api-server-host", "", "Host cluster API server HOST")
+	flags.StringVar(&o.HostAPIServerPort, "host-api-server-port", "", "Host cluster API server PORT")
```

Comment:
```suggestion
	flags.StringVar(&o.HomeAPIServerHost, "home-api-server-host", "", "Override the home cluster API server HOST")
	flags.StringVar(&o.HomeAPIServerPort, "home-api-server-port", "", "Override the home cluster API server PORT")
```

We usually call "home" the originating cluster and "remote" the target one; I would use the same naming here and in the other occurrences.

And additionally, I would update the flag description to clarify that this is not required but is an override of the default value.